codalab/rest/account.py

```diff
@@ -78,7 +78,7 @@ def do_signup():
     if not token:
         errors.append('Google reCAPTCHA token is missing.')
     else:
-        url = 'https://www.google.com/recaptcha/api/siteverify'
+        url = 'https://www.recaptcha.net/recaptcha/api/siteverify'
```

Comment:
What is https://www.recaptcha.net? Do you have an info page on what it's used for / who runs it / how it uses data?

Reply:
Oh I found it, it's google. Can you link to https://developers.google.com/recaptcha/docs/faq#can-i-use-recaptcha-globally in the comments?